.github/workflows/ci.yml

```diff
@@ -24,27 +24,27 @@ jobs:
       matrix:
         os: [ubuntu-latest, macos-latest]
         scala: [2.12.15]
-        java: [temurin@11, graal_20.3.1@11]
+        java: [temurin@17, graal_22.3.1@17]
     runs-on: ${{ matrix.os }}
     steps:
       - name: Checkout current branch (full)
         uses: actions/checkout@v3
         with:
           fetch-depth: 0
 
-      - name: Setup Java (temurin@11)
-        if: matrix.java == 'temurin@11'
+      - name: Setup Java (temurin@17)
+        if: matrix.java == 'temurin@17'
         uses: actions/setup-java@v3
         with:
           distribution: temurin
-          java-version: 11
+          java-version: 17
 
-      - name: Setup GraalVM (graal_20.3.1@11)
-        if: matrix.java == 'graal_20.3.1@11'
+      - name: Setup GraalVM (graal_22.3.1@17)
+        if: matrix.java == 'graal_22.3.1@17'
         uses: DeLaGuardo/setup-graalvm@5.0
```

Comment:
This should be replaced by the official action.
https://github.com/graalvm/setup-graalvm

Reply:
Thanks, will do.

Reply:
I have a PR for that here - https://github.com/sbt/sbt-github-actions/pull/140

Reply:
Okay so I would recommend merging https://github.com/sbt/sbt-github-actions/pull/140 first and then I can rebase this PR.